answers/Mikhail/stackoverflow/google/google-search/google-ad/google-add-titles/google-ad-titles-scrape-cheerio.js

```diff
@@ -0,0 +1,34 @@
+const unirest = require("unirest");
```

Comment:
```suggestion
// https://stackoverflow.com/questions/70853810/returned-null-response-from-unirest-block
// https://replit.com/@MikhailZub/Google-Ad-Results-with-NodeJS-SerpApi#index.js

const unirest = require("unirest");
```

Link to an answer and replit. This is helpful if you need to answer a similar question.